CliClient/tests/models_Setting.js

```diff
@@ -31,4 +34,51 @@ describe('models_Setting', function() {
 		expect('username' in output).toBe(false);
 	}));
 
+	it('should save secure items', asyncTest(async () => {
+		const originalScheduleSave = Setting.scheduleSave;
+		const originalSaveSecureItem = shim.saveSecureItem;
+
+		Setting.scheduleSave = async () => {
+			Setting.saveTimeoutId_ = 123;
+			await Setting.saveAll();
+		};
```

Comment:
Why did you need to override this method?

Reply:
I had to override this method so that when I called `Setting.setValue(key, value)`, the key-value pair is immediately saved. Otherwise, the save is performed asynchronously and I couldn't check if the values were correctly saved.

Reply:
Right, that makes sense. However you can simply set the value and then call `await Setting.saveAll()` just after, and they will be saved them immediately. Could you make this change please?